site/content/blog/release-v1.13.0.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Release v1.13.0 - Magefiles Directory"
+date = 2022-03-18T08:17:52-05:00
+type = "post"
+author = "Nate Finch"
+authorLink = "twitter.com/natethefinch"
++++
+
+[Mage v1.13.0](https://github.com/magefile/mage/releases/tag/v1.13.0) is now
+released with a few fixes and one big new feature - the magefiles subdirectory.
+
+You can now put your magefiles in a subdirectory of your repo called
+`/magefiles`, and Mage will find them automatically, like it always has for
+files in the root. Files in `/magefiles` do not need the `mage` build tag to be
+discovered by Mage (but if it’s there, it’ll still find them).
+
+This is a big change that will be a significant quality of life improvement for
+many developers. Until now, Mage was designed to allow you to keep magefiles in
+the root of your repo, tagged with the `mage` build tag, so that your normal go
+build tooling would ignore them. However, this had the side effect of meaning
+that linting etc would *also* ignore them. You could add the `mage` build tag to
+your IDE and/or gopls, but because Mage also requires the files to be in
+`package main`, this could result in having files from two different packages in
+the root of your repo, which would cause linters and gopls to complain.
+
+Now, you can put those files in a separate directory, and you don’t even need
+the build tag any more. No conflicting packages, no need to twiddle with build
+tags, and all this is automatic, no settings need to be adjusted, you just need
+Mage v1.13+.
+
+Migrating your magefiles from the root of your repo to a subdirectory is
+trivial. Just copy them into `/magefiles`.  That’s it. Mage will continue to
```

Comment:
Should this say "Just move them" instead of "Just copy them"?

Reply:
Yes. I often say copy when I mean move, for some reason.